RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/Playback/PlaybackScreen.java

```diff
@@ -463,8 +538,27 @@ public void onClick(View v) {
                     undo();
                     break;
                 }
-                case R.id.btnMarkerMode: {
-                    System.out.println("VERSE MARKER MODE");
+                case R.id.btnEnterVMarkerMode: {
+                    enterVMarkerMode();
+                    break;
+                }
+                case R.id.btnExitVMarkerMode: {
+                    exitVMarkerMode();
+                    break;
+                }
+                case R.id.btnDropVMarker: {
+                    dropVMarker();
+                    mVerses -= 1;
```

Comment:
should mVerses be renamed to mVersesToMark?
decrementing mVerses as markers are placed seems counter-intuitive